backend/src/pong/Game.types.ts

```diff
@@ -0,0 +1,30 @@
+export class Position {
+	x: number;
+	y: number;
+}
+
```

Comment:
are these types specially for the backend otherwise why are there two games.ts files?

Reply:
Is needed as the type files are in two different containers... we can maybe do a symlink, not sure if this copies it to the both docker files though....? Do you know?

Reply:
For now renamed them with either a .fe or .be

Reply:
cool